### PR TITLE
ImageRenderingContext

### PR DESCRIPTION
**Pull request by pelikhan**
- [x] minimal canvas rendering context implementation, no rotation, scale. Mostly to have well know apis.

### CODE REVIEW

File: packages/graphics/src/image.ts [hunk 1]
```diff
@@ -251,6 +257,10 @@ Image.prototype.rotated = function (deg: number): Image {
     }
 }
 
+Image.prototype.getContext = function (): ImageRenderingContext {
+    return new ImageRenderingContext(this)
```
Comment (mmoskal) [inline packages/graphics/src/image.ts:261]: we should either rename this `createContext()` or have it return the same value after construction as the web version does...

Comment (mmoskal) [reply packages/graphics/src/image.ts:261]: (I would be inclined to rename, more flexible and memory efficient)

**Comment by pelikhan** [comment]
[like]  Peli de Halleux reacted to your message:
________________________________

File: packages/graphics/src/image.ts (hunk 1 above)
Comment (pelikhan) [reply packages/graphics/src/image.ts:261]: I can, name?

**Pull request merged**